pages/windows/invoke-item.md

```diff
@@ -0,0 +1,29 @@
+# Invoke-Item
+
+> Open files in their respective default programs.
+> This command can only be used through PowerShell.
+> More information: <https://docs.microsoft.com/powershell/module/microsoft.powershell.management/invoke-item>.
+
+- Open a file in its default program:
+
+`Invoke-Item -Path "{{path/to/file}}"`
+
+- Open all files inside a directory:
+
+`Invoke-Item -Path "{{path/to/directory}}\*"`
+
+- Open all PNG files inside a directory:
+
+`Invoke-Item -Path "{{path/to/directory}}\*.png"`
```

Comment:
```suggestion
`Invoke-Item -Path "{{path\to\directory\*.png}}"`
```

The entire argument should be wrapped in brackets.

Question for other maintainers:
Since this is Windows-specific, should we be using forward slashes or backslashes? 🤔 At the very least, we probably don't want to mix between both of them in the same argument.

Reply:
In most cases, Windows will automatically convert these slashes into backslashes (either in Command Prompt or PowerShell) so `C:/Users/User/Documents` is still valid, too.

In most DOS and non-PowerShell commands, command parameters (e.g. `/d`) do use forward slashes but only when the character is inserted at the begining of the sentence. For the case of root directory, users may end up to use `C:/` (e.g. `cd C:/`) instead of `/` (e.g. `cd /`) which should be okay.

Reply:
@SethFalco the `/` is full compatible with Windows, and the `/` is used in the common platform. To keep the same format `/` should be used in the Windows platform.
```suggestion
`Invoke-Item -Path {{path/to/directory/*.png}}`
```
Also, we don't use `"` for paths.

Reply:
Ref quoting the path: if you want the glob to work IIRC you need to remove the quotes, yeah